### ISSUE TITLE
Composition and semantics

### ISSUE THREAD

**Issue by MattReimer** [react-cosmos/react-cosmos#608]
This may be more of a philosophical thing. I have a form I'm designing in `semantic-ui-react` and one of the main things I'm trying to test is how everything fits inside a modal dialog. 

The modal dialog is a separate container component I reuse a bunch of places so my JSX looks something like:

```jsx
<ModalContainer>
   <UnitsForm />
</ModalContainer>
```

So the fixture I end up with looks like this:

```js
import React from 'react'
import UnitsForm from '../UnitsForm'
import FormModal from '../ModalContainer'

export
default {
  component: ModalContainer,
  props: {
    children: <UnitsForm />
  }
}
```

There's two problems I see with this:

1. The component I'm testing isn't really `<ModalContainer>` so this feels a little wrong somehow. 
2. When I change `<UnitsForm>` my browser doesn't pick up the change. Must be something to do with it being a child component? 

I'm new here so I'm happy if the answer to this is simply "don't do that" but I couldn't find a better way in the docs or issues I've read so far. Suggestions?

**Comment by skidding** [comment]
I agree, it would be more natural for the fixture to look like this:

```js
export default {
  component: UnitsForm,
  decorator: ModalContainer
}
```

This feature is not yet available, but it's discussed in #568 and I'm more than happy to provide guidance if anyone is willing and has the time to contribute.

**Comment by MattReimer** [comment]
Ah, yes. That would be cool. We can continue this conversation on #568 for sure.
